src/utils/createPptx/pptxConfiguration/masterSlide.ts

```diff
@@ -10,52 +10,6 @@ import {
-// const IMAGE_PATH = '/public/logo.png'
-
-export const masterDescriptionSlide: PptxGenJS.SlideMasterProps = {
```

Comment:
Was this not used?

Reply:
No. I search for it throughout the whole project. but it was only referred to in that file